docs/ABI/Mangling.rst

```diff
@@ -153,6 +152,10 @@ Globals
   global ::= entity 'Wvd'                // field offset
   global ::= entity 'WC'                 // resilient enum tag index
 
+  global ::= nominal-type 'MU'           // ObjC metadata update callback function
+  global ::= nominal-type 'Ms'           // ObjC resilient class stub
+  global ::= nominal-type 'Mt'           // Full ObjC resilient class stub (private)
```

Comment:
Why are these down here instead of up next to the other `M` manglings where somebody looking for an available mangling would probably be looking?

Reply:
Ok, I'll put it back.